frontend/src/scenes/insights/Histogram/Histogram.tsx

```diff
@@ -260,5 +260,5 @@ export function Histogram({
         [data, config]
     )
 
-    return <div className="histogram-container" ref={ref} style={{ minWidth: config.width }} />
```

Comment:
:+1: Not a fan of this `config`-based rendering… It's like a "god" object, makes things harder to adjust.

Reply:
Not a fan either, but irrc it was required to enforce d3's width calculations on the div wrapping the svg so that scrolling horizontally worked.